src/app/locale/fr_FR/FireGento_MageSetup.csv

```diff
@@ -24,7 +24,7 @@
 "Update System Config Settings","Mettre à jour les réglages de la configuration système"
 "i.e. newsletter confirmation, account confirmation, ..", "expl. confirmation de newsletter, confirmation de création de compte, ..."
 "Create CMS Pages and Blocks","Créer les pages CMS et blocs"
-"i.e. shipping, conditions, revocation","expl. livraisons, conditions, révocation"
+"i.e. shipping, conditions, revocation","expl. livraisons, conditions, rétractation"
```

Comment:
@jacquesbh Thanks for the correction, of course you are right. I want to note that these are words I didn't touch at all (been wrong before). Currently I am busy with another project, can somebody please accept this PR, and commit the proposed changes for jacquesbh?